script/sccache

```diff
@@ -6,30 +6,28 @@ import sys
 
 import lib.sccache as sccache
 
+
 def parse_args():
   parser = argparse.ArgumentParser(description='Run sccache`')
+
   parser.add_argument('--azure_container', help='Name of azure container to use')
   parser.add_argument('--azure_connection', help='Name of azure connection to use')
```

Comment:
Can we get rid of all argument parsing and just pass the args straight through to sccache? I don't think these args are necessary, all they do is set an environment variable.

Reply:
@jkleinsc ^

Reply:
@alexeykuzmin @nornagon those args are needed for VSTS because they are stored in secret variables that can't be accessed directly as environment variables but they can be passed along in a command invocation, but sccache doesn't support them that way.

Reply:
Can they not be passed like:

```sh
AZURE_CONTAINER=$azure_container_secret_variable ninja out/Default ...
```
?

Reply:
Unfortunately, no: https://docs.microsoft.com/en-us/vsts/pipelines/process/variables?view=vsts&tabs=designer%2Cbatch#set-in-script